pkg/engine/jmespath/functions.go

```diff
@@ -834,6 +834,14 @@ func jpLabelMatch(arguments []interface{}) (interface{}, error) {
 	return true, nil
 }
 
+func ToBoolean(input string) bool {
```

Comment:
You probably want to return an error too.

Reply:
Now it's, resolved.